nightguard Widget Extension/SystemSmallView.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  SystemSmallView.swift
+//  nightguard Widget Extension
+//
+//  Created by Philipp Pöml on 26.05.24.
+//  Copyright © 2024 private. All rights reserved.
```

Comment:
This should be removed, since it clashes with the nightscout license.

Reply:
Sure, this is something Xcode buts into every file, like the whole project is full of it:
```
//
//  ViewExtension.swift
//  nightguard
//
//  Created by Dirk Hermanns on 08.04.23.
//  Copyright © 2023 private. All rights reserved.
//
```
But you can put your name, if that is better.

Reply:
Yes - I know. But should be omitted for new PRs.